scripts/hps-mc-construct-detector.in

```diff
@@ -0,0 +1,143 @@
+#!/usr/bin/env bash
```

Comment:
One thing we should think about if we are writing a general script like this. org.hps.detector.DetectorConverter is gonna look for a directory called gdml and won't work unless it finds it and the correct resources in that directory. We could have this script be smart enough to make a link if needed, but we would need to tell it where the hps-java source code is to make that link.

Reply:
Yea, right now I deduce where hps-java is by having the user provide the full path to the detector directory. But that is kinda ugly - is there a env variable for the location of hps-java?

Reply:
The way we handle this for [SLIC] where this same thing is also an issue is by adding a config to the ~/.hpsmc called `detector_dir`

```
[SLIC]
slic_dir = /sdf/group/hps/users/bravo/src/slic/install
hps_fieldmaps_dir = /sdf/group/hps/users/bravo/src/hps-fieldmaps
detector_dir = /sdf/group/hps/users/bravo/src/hps-java/detector-data/detectors
```

Reply:
if this script is called via the Job Component, it could just pass it to this script as an arg